crates/context/config/src/client/env/config/query/application.rs

```diff
@@ -161,9 +160,8 @@ impl Method<Evm> for ApplicationRequest {
 
     fn encode(self) -> eyre::Result<Vec<u8>> {
         let context_id: [u8; 32] = self.context_id.rt().expect("infallible conversion");
-        let context_id_val = B256::from_slice(&context_id);
 
-        Ok(SolValue::abi_encode(&(context_id_val)))
+        Ok(SolValue::abi_encode(&(context_id)))
```

Comment:
why not this?

```suggestion
        Ok(context_id.abi_encode())
```

Reply:
Updated all queries to this, initial version was following already existing encoding style.